specification/2.0/Makefile

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020 The Khronos Group Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Output specification targets
+all: allhtml allpdf
+
+allhtml: README.html SingleSpec.html
+
+allpdf: README.pdf SingleSpec.pdf
+
+# Generate the Properties Reference section of the spec from JSON schema.
+# This assumes the 'wetzel' repository is cloned in a directory parallel
+# to the glTF repository.
+WETZELROOT = ../../../wetzel
+WETZEL = $(WETZELROOT)/bin/wetzel.js
+# Base name of the generated properties reference file
+PROPREF = PropertiesReference
+$(PROPREF).adoc: $(WETZEL) $(wildcard schema/*.json)
+	$(WETZEL) -l2 -a=cqo -m=a -p schema \
```

Comment:
Hi Jon, when you get a chance, pull the latest wetzel changes, and update this Makefile.

- Replace `-l2` with `-n` (Get rid of Level 2, moving up to Level 1 by default, and use new "No TOC" option)

- Replace `-p schema` with `-p 'https://github.com/KhronosGroup/glTF/tree/master/specification/2.0/schema'`

We may yet need to bring the contents of the schema files themselves in here, for ISO's sake...

Reply:
(xrefs are not done yet, still pending...)

Reply:
Done. The submodule notion is looking more attractive by the minute.

BTW, I'm not familiar with the npm package management system. How can I update needed packages in package.json without messing it up in the process? I was just installing by hand e.g. 'npm install minimist' but then that forced me to update to a later version than what was in package.json and modified package.json in the process, which caused a conflict when I tried pulling the new wetzel.

Reply:
Also, if you want you're welcome to propose PRs targeting this branch while it's around, or I think there's a way to propose edits inline in the existing PR page though haven't used it.

Reply:
> How can I update needed packages in package.json without messing it up in the process?

Ah, revert `package.json` and then run `npm install` in the wetzel root folder.  The packages end up in the `node_modules` folder at the top level of the repo, and ideally git should not find any changes as a result.

We'll do the submodule thing after this branch is merged.  I think it's too much trouble for folks who have to switch between branches, causing the whole submodule to appear and disappear.